app/views/exchanges/hbx_profiles/outstanding_verification_datatable.html.erb

```diff
@@ -3,7 +3,7 @@
 <p id="notice"><%= show_table_notice(notice) %></p>
 
 <div class='container'>
-  <h1> Outstanding Verifications </h1>
+  <h1><%= l10n('hbx_profiles.outstanding_verifications.header') %></h1>
```

Comment:
any concern not flagging this?

Reply:
This think is fine since just a refactor